pkg/webui/console/containers/fetch-table/index.js

```diff
@@ -89,6 +90,58 @@ class FetchTable extends Component {
     this.debounceCancel = cancel
   }
 
+  static propTypes = {
+    addMessage: PropTypes.message,
+    fetching: PropTypes.bool,
```

Comment:
`baseDataSelector` should also be here.

Reply:
eslint doesn't recognize baseDataSelector as a propType, which means it won't check for it so there is no purpose in defining it I believe. I added it but not sure if its necessary.

Reply:
Weird? I can set it with no warning
```js
…
baseDataSelector: PropTypes.shape({
items: base[props.entity] || [],
    totalCount: PropTypes.number,
    fetching: PropTypes.bool,
    fetchingSearch: PropTypes.func,
    pathname: PropTypes.string,
    mayAdd: PropTypes.bool,
}).isRequired,
…